utils/functions.py

```diff
@@ -277,17 +266,17 @@ def paginate(iterable, n, fillvalue=None):
     return [i for i in zip_longest(*args, fillvalue=fillvalue) if i is not None]
 
 
-async def get_selection(ctx, choices, delete=True, return_name=False, pm=False, message=None):
+async def get_selection(ctx, choices, delete=True, pm=False, message=None, force_select=False):
```

Comment:
removed the `return_name` kwarg and added `force_select`
- the former was not used anywhere and not well-documented
- the latter forces a dialog, even if there's only one choice